gpt4all-bindings/rust/gpt4all-sys/wrapper.h

```diff
@@ -0,0 +1 @@
+#include <llmodel_c.h>
```

Comment:
The file's missing a terminating newline. Otherwise this should be fine, but not knowing Rust I'm left wondering:

- Is this header even required anymore? Couldn't you include llmodel_c.h directly from wherever wrapper.h is currently used?
- Where is the include path being defined to actually find llmodel_c.h?

Reply:
I expect the headers to be somewhere along the users include path. Finding the .so files is similar. 

I considered finding both by utilizing the fact the crate is in the same repo as the project it is binding (at one iteration the build script invoked cmake). Most `sys` crates in rust attempt to bind the system version of a library instead of bringing their own, but it's not uncommon to build your own for less common libraries. 

I made the PR to defer these kinds of decisions, would you prefer the project builds it's down gpt4all backend and links to that? Or should I add more documentation on building your own? Having both is also an option which would be controlled through the crates feature flag. What I've built here is the smallest possible useful amount of code.

Reply:
I don't know what the preferred way is to build the full project (the backend and bindings), someone else should comment on that. On the other hand I'm pretty sure that llmodel_c.h is probably not comparable to system library headers. Maybe someone else could weigh in at this point?